.github/workflows/deployment.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         with:
           repository: AltaPay/plugin-infrastructure
           token: ${{ secrets.ACCESS_TOKEN }}
-          ref: 'origin/main'
+          ref: 'origin/magento_composer_install'
```

Comment:
Is this correct? We should always depend on `main` here.

Reply:
Yes, should be `main`. I did not know the task was ready for review, so, temporarily changed it for debugging.